### ISSUE TITLE
Regarding the division of the "scared" dataset mentioned in the paper (15351, 1705, and 551 frames for the training, validation and test sets, respectively)

### ISSUE THREAD

**Issue by lijunfei-stu** [ShuweiShao/AF-SfMLearner#67]
thank you for your work. I have one question. After I completed processing the SCARED dataset using the scared_toolkit tool (only processing datasets 1, 2, 3, 5, 7, 8, and 9), I found that the training set (1, 2, 3, 5, 7) had a total of 17,212 pairs of stereo images, and the test set (8, 9) had a total of 5,909 pairs of stereo images. In your paper description, it was mentioned that "split the SCARED dataset into 15351, 1705, and 551 frames for the training, validation and test sets, respectively". I would like to know if you have performed any other processing on the dataset and if some data needs to be excluded?
Once again, thank you for your hard work. I look forward to your reply!
